### DISCUSSION TITLE
Webhook active, webhook-responses kosong

### DISCUSSION THREAD

**Discussion by sorryjack** [kirimwa-id/forum#176] [Tanya Jawab]
Webhook sudah diset, incoming message ada, tapi outgoing message seperti tidak menerima webhook, cek di get/webhook-responses juga kosong. Tolong bantuannya gaisss...

**Comment by sorryjack** [comment]
Saya juga sudah test dengan webhook.site namun tidak ada request meski wa sudah terkirim. mohon bantuannya mas rio

**Comment by agiltriyas** [comment]
sudah solve mas yang ini ? boleh info gmn caranya , terimakasih

**Comment by Fajar-G** [comment]
kalau saya kasusnya Incoming Message yang engga keterima, tapi Message Response bisa. webhook-responses ada isinya terakhir tanggal 2024-02-18. apa yang salah ya?
